docs/tutorials/tutorial_siesta_1.ipynb

```diff
@@ -250,7 +250,10 @@
     "    h2o.plot(axes=\"xy\", atoms_style={\"size\": normalize(n), \"color\": c})\n",
     "    for n, c in zip(\n",
     "        h2o.apply(\n",
-    "            es.norm2(sum=False), np.sum, mapper=partial(h2o.a2o, all=True), axis=1\n",
+    "            es.norm2(projection=\"orbital\"),\n",
+    "            np.sum,\n",
+    "            mapper=partial(h2o.a2o, all=True),\n",
```

Comment:
This mapper is doing what could now also be achieved with `projection="atom"`, or?

Reply:
Yes, these two plots can now simply be achieved with
```python
plots = [
    h2o.plot(axes="xy", atoms_style={"size": normalize(n), "color": c})
    for n, c in zip(es.norm2(projection="atom"), ("red", "blue"))
]
```

Reply:
Great! Yes!